ckanext/recombinant/templates/recombinant/resource_edit.html

```diff
@@ -20,11 +20,14 @@
 {% block primary_content %}
   {% if dataset %}
     <ul class="nav nav-pills mrgn-bttm-sm">
+      {% set separator = ' : ' if h.lang() == 'fr' else ': ' %}
       {% for r in dataset.resources %}
+        {% set num_label = separator ~ ngettext("%(num)d "+_('row'), "%(num)d "+_('rows'), r.get('datastore_rows', 0)) | format(num=r.get('datastore_rows', 0)) %}
```

Comment:
This isn't how you ngettext. babel won't be able to extract these strings.

Reply:
@wardi I just copied from whatever was there already. What is the best way to do this in jinja?

Reply:
missed it earlier then. Better to fix in both places then. It should look like https://github.com/ckan/ckan/blob/b0ed536f5c6611522f2708a4babb881075dfc3a6/ckan/templates/organization/manage_members.html#L15

Reply:
This is done now, strings have been extracted and compiled